cli/src/semgrep/formatter/sarif.py

```diff
@@ -162,23 +217,20 @@ def _dataflow_trace_to_codeflow_sarif(
 
         # TODO: handle rule_match.taint_sink
         if isinstance(taint_source.value, out.CliCall):
-            logger.error(
-                "Emitting SARIF output for unsupported dataflow trace (source is a call)"
-            )
-            return None
+            location = taint_source.value.value[0][0]
+            code_flow_message = f"Untrusted dataflow from {str(location.path.value)}:{str(location.start.line)} to {str(rule_match.path)}:{str(rule_match.start.line)}"
```

Comment:
This line is in both branch of the `if`
Please move it outside to avoid duplication

Reply:
Done